### PR TITLE
v0.6.0 release PR

### PR DESCRIPTION
**Pull request by adithyaov**

### CODE REVIEW

**Comment by adithyaov** [comment]
See #60 for every CI uncommented.

File: .packcheck.ignore [hunk 1]
```diff
@@ -1 +1,9 @@
 .packcheck.ignore
+stack.yaml
+.ci/stack-8.0.yaml
+.travis.yml
+.circleci/config.yml
+.github/workflows/packcheck.yml
+appveyor.yml
```
Comment (harendra-kumar) [inline .packcheck.ignore:7]: This is a package for showing example configuration and for CI and it has ready made configs for popular CI platforms. So these files should be packaged in the distribution.

I think we should package all these files and even stack.yaml/cabal.project etc.

**Review by harendra-kumar — CHANGES_REQUESTED** [review]
The README can be updated to mention the github CI support as well.

**Review by harendra-kumar — COMMENTED** [review]
1. Regarding the github CI matrix. A CI matrix that multiplies multiple lists is usually fraught with problems. We do not always want to test every dimension crossed with every other dimension. That makes too many tests. And also, if we add one test in one dimension that leads to addition of many tests.
2. Create a v0.6.0 tag and use that tag for PACKCHECK_GITHUB_COMMIT. The config files shipped with the package should have that tag.

**Comment by harendra-kumar** [comment]
We can replace `cabal-v2` with cabal in the README. Also, the "out of the box support" section in the readme does not mention github.

**Comment by harendra-kumar** [comment]
It mentions github, I looked at the wrong readme.

**Review by harendra-kumar — APPROVED** [review]

**Pull request merged**